### PR TITLE
Clarify how to install the extension via Tableau Exchange

### PR DESCRIPTION
**Pull request by bcantoni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
-# Using the Extension
+## Using the Extension
 1. Set your dashboard filters they way you want to be able to revert to.
 2. Open the configuration window and click "Save Settings".
 3. Optional: Customize your buttons label and colors.
 
 Note: You can add as many instances of this extension as you like!
 
-## How to install for local use
+## Building for local use
```
Comment (jacalata) [inline README.md:16]: Is there any scenario where they want to install it locally and NOT update any code? Or can we make this heading slightly clearer/more offputting like 

1. Using the Extension from the Tableau Gallery (recommended)
a. set your dashboard....


2. Download the Extension code to develop locally
a...

Comment (bcantoni) [reply README.md]: Good point - I think it's just the 2 modes, either use from Exchange or DIY. I pushed an update per your wording advice.

Comment (bcantoni) [reply README.md]: image

File: README.md [hunk 2]
```diff
@@ -23,4 +27,6 @@ Note: You can add as many instances of this extension as you like!
 6. Update the existing or create a new manifest file (.trex) to point to the URL where you are hosting the extension with `/#/bookmarks` at the end. For example: `http://localhost:8080/#/bookmarks`.
 
 ## Support
-If you have questions about the extension or found a bug please open a new [issue](https://github.com/tableau/extension-filter-bookmarks/issues).
+Paid Tableau users can contact the Tableau Technical Support team for help.
```
Comment (stephendeoca) [inline README.md:30]: I'm not sure what a "Paid Tableau users" is? Do you mean Tableau users with a non free license? If there's a way to become a "Paid Tableau user" could you possibly link that as well?

Comment (bcantoni) [reply README.md:30]: Maybe simpler to just say "Tableau customers can contact...", wdyt?

It just means people who are paying for Tableau can use our tech support.

Comment (stephendeoca) [reply README.md:30]: "Tableau customers can contact" sounds better. Do you happen to know if customers such as students that get a free license be included for tech support access? If not, it should be still fine.

Comment (bcantoni) [reply README.md]: No it's just paid customers who can use technical support. I've updated the description with a new commit.

**Review by stephendeoca — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (stephendeoca) [reply README.md]: Looks good

**Pull request merged**